lib/web/ui/usertask.go

```diff
@@ -99,44 +99,56 @@ func MakeUserTasks(uts []*usertasksv1.UserTask) []UserTask {
 
 // MakeDetailedUserTask creates a UI UserTask representation containing all the details.
 func MakeDetailedUserTask(ut *usertasksv1.UserTask) UserTaskDetail {
-	var title string
-	var description string
-
 	var discoverEC2 *usertasks.UserTaskDiscoverEC2WithURLs
 	var discoverEKS *usertasks.UserTaskDiscoverEKSWithURLs
 	var discoverRDS *usertasks.UserTaskDiscoverRDSWithURLs
 
 	switch ut.GetSpec().GetTaskType() {
 	case apiusertasks.TaskTypeDiscoverEC2:
-		title, description = usertasks.DescriptionForDiscoverEC2Issue(ut.GetSpec().GetIssueType())
 		discoverEC2 = usertasks.EC2InstancesWithURLs(ut)
 
 	case apiusertasks.TaskTypeDiscoverEKS:
-		title, description = usertasks.DescriptionForDiscoverEKSIssue(ut.GetSpec().GetIssueType())
 		discoverEKS = usertasks.EKSClustersWithURLs(ut)
 
 	case apiusertasks.TaskTypeDiscoverRDS:
-		title, description = usertasks.DescriptionForDiscoverRDSIssue(ut.GetSpec().GetIssueType())
 		discoverRDS = usertasks.RDSDatabasesWithURLs(ut)
 	}
 
+	_, description := userTaskTitleAndDescription(ut)
```

Comment:
Is the description generally big? Is that why we're not including it in `UserTask` like the title?

Reply:
My first implementation didn't include a description or title in the List endpoint.

However, we need to show the title when listing (see the screenshot in the PR description).

So, I've only added the title when listing.
Description is indeed much longer and also only used when viewing a single user task. So, I think we should only include it in the `GET /usertasks/<task-id>` endpoint.